packages/istanbul-reports/lib/html/assets/base.css

```diff
@@ -172,16 +172,17 @@ table.coverage td span.cline-any {
 /* dark green */
 .status-line.high, .high .cover-fill { background:rgb(77,146,33) }
 .high .chart { border:1px solid rgb(77,146,33) }
-
-.medium .chart { border:1px solid #666; }
-.medium .cover-fill { background: #666; }
+/* dark yellow (gold) */
+.status-line.medium, .medium .cover-fill { background: #f9cd0b; }
```

Comment:
This seems to me like a 'bright yellow' rather than dark yellow.  How would you feel about `#f1c50b` instead (for `.medium .chart` as well)?